test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -149,13 +150,28 @@ upgrade_downgrade()
         done
 }
 
+
+if [ $# -eq 1 ]; then
+    if [ $1 = "-l" ]; then
```

Comment:
`-f` feel more intuitive to me (compared to `-l`), but I do not have a strong feeling about that.